modules/ztp-hub-factory-pipeline.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="hub-factory-pipeline_{context}"]
+= The hub factory pipeline
+
+The hub configuration pipeline stage prepares the hub cluster to deploy multiple edge clusters for the end customer.
+
+.Prerequisites
+
+* An {product-title} hub cluster available with {product-version} installed.
+* The correct DNS entries and the API and ingress exposed in the factory network (DHCP external network).
+* At least 6 persistent volumes available and ready to be bound.
+* Internet connectivity.
+* OpenShift Pipelines already created.
+
+
+The flow associated with deploying the hub cluster is:
+
+Check hub::
+
+The initial stages in the hub pipeline downloads the various tools needed. It downloads `jq`, `oc`, `opm` and `kubectl`. It also proceeds to verify that various hub install prerequisites exist before proceeding, for example it checks the:
+
+* {product-title} version.
+* Nodes are ready.
+* Cluster Operators are ready.
+* Metal^3^ pods are ready.
```

Comment:
If we use Metal^3^, let's be consistent elsewhere where we use Metal3.  Either is fine, but would be good to be consistent also in the diagrams.

Reply:
Good point will revert to Metal3 as that is consistent with diagram